requirements.txt

```diff
@@ -44,5 +45,5 @@ social_auth_app_django==4.0.0
 tomd==0.1.3
 wagtail-cache==1.0.1
```

Comment:
It would be good to check if other Wagtail specific packages are compatible with the latest Wagtail 2.13 version.

```shell
wagtail-cache==1.0.1
wagtail-purge==0.1
```

Reply:
Updated wagtail-cache to latest version.